src/column_family.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 Tyler Neely
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::marker::PhantomData;
+
+use crate::{Options, DB};
+
+/// A descriptor for a RocksDB column family.
+///
+/// A description of the column family, containing the name and `Options`.
+pub struct ColumnFamilyDescriptor {
+    pub(crate) name: String,
+    pub(crate) options: Options,
+}
+
+impl ColumnFamilyDescriptor {
+    // Create a new column family descriptor with the specified name and options.
```

Comment:
It's a comment now. I suppose you've meant doc comment ? 
```suggestion
    /// Create a new column family descriptor with the specified name and options.
```